lib/parser.js

```diff
@@ -222,7 +222,8 @@ Parser.prototype._parseBlockElements = function(indexApiBlocks, detectedElements
                     preventGlobal = elementParser.preventGlobal === true;
 
                     // allow multiple inserts into pathTo
-                    allowMultiple = elementParser.allowMultiple === true;
+                    //allowMultiple = elementParser.allowMultiple === true;
```

Comment:
no need to keep the old code commented ;)

Reply:
i have removed it now